doc/development/RELEASING.md

```diff
@@ -1,9 +1,147 @@
 # Releasing
 
-_Release process documentation is in progress, see [PR 5252](https://github.com/bundler/bundler/pull/5252)._
+Bundler users [Semantic Versioning](https://semver.org/).
+
+_Note: In the documentation listed below, the *current* minor version number is
+1.11 and the *next* minor version number is 1.12_
+
+Regardless of the version, *all releases* must update the `CHANGELOG.md` and `lib/bundler/version.rb`
+files. The changelog for the first stable minor release (`1.12.0`) is a sum of all
+the preceding pre-release versions (`1.12.pre.1`, `1.12.pre.2`, etc) for that
+minor version. The changelog for the first stable minor release is left blank
+unless there are fixes included since the last pre/rc release.
+
+## Workflow
+
+In general, `master` will accept PRs for:
+
+* feature merges for the next minor version (1.12)
+* regression fix merges for a patch release on the current minor version (1.11)
+* feature-flagged development for the next major version (2.0)
+
+### Breaking releases
+
+Bundler cares a lot about preserving compatibility. As a result, changes that
+break backwards compatibility should (whenever this is possible) include a feature
+release that is backwards compatible, and issue warnings for all options and
+behaviors that will change.
+
+We try very hard to only release breaking changes when incrementing the _major_
+version of Bundler.
+
+### Cherry picking
+
+Patch releases are made by cherry-picking bug fixes from `master`.
+
+When we cherry-pick, we cherry-pick the merge commits using the following command:
+
+```bash
+$ git cherry-pick -m 1 MERGE_COMMIT_SHAS
+```
+
+For example, for PR [#5029](https://github.com/bundler/bundler/pull/5029), we
+cherry picked commit [dd6aef9](https://github.com/bundler/bundler/commit/dd6aef97a5f2e7173f406267256a8c319d6134ab),
+not [4fe9291](https://github.com/bundler/bundler/commit/4fe92919f51e3463f0aad6fa833ab68044311f03)
+using:
+
+```bash
+$ git cherry-pick -m 1 dd6aef9
+```
+
+The `rake release:patch` command will automatically handle cherry-picking, and is further detailed below.
+
+## Changelog
+
+Bundler maintains a list of changes present in each version in the `CHANGELOG.md` file.
+Entries should not be added in pull requests, but are rather written by the Bundler
+maintainers in the [bundler-changelog repo](https://github.com/bundler/bundler-changelog).
+That reposity tracks changes by pull requests, with each entry having an associated version,
+PR, section, author(s), issue(s) closed, and message.
+
+Ensure that repo has been updated with all new PRs before releasing a new version,
+and copy over the new sections to the `CHANGELOG.md` in the Bundler repo.
```

Comment:
How does one update that repo? I honestly don't remember, and that repo doesn't have a readme. 😭

Reply:
uh, I haven't done so since my last release. want to open an issue there for someone to add a README ?